pyo3-macros-backend/src/pyclass.rs

```diff
@@ -512,6 +531,57 @@ fn impl_enum(
     impl_enum_class(enum_, args, doc, methods_type, krate)
 }
 
+fn impl_into_py_enum(cls: &syn::Ident, variants: &[PyClassEnumVariant<'_>]) -> TokenStream {
+    // Assuming all variants are unit variants because they are the only type we support.
+    match variants.len() {
+        0 => quote! {},
+        1 => {
```

Comment:
What to do with the "no variants" case? It should never even reach here as it errors earlier already on `#[pyclass] can't be used on enums without any variants`. 

Though maybe it's not needed to branch on the number of variants. I believe that the general case could/should also work when there is just a single variant. But this is my first time writing macros and I could not get it to work.